.nais/deploy-sandbox.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: pensjonskalkulator-frontend-sandbox
```

Comment:
Denne trengs egentlig ikke endres? Den heter 'pensjonskalkulator-frontend' både for dev og prod ser jeg.

Reply:
Jo, det er navn på appen i namespace. To apper kan ikke kjøre med samme navn i det samme namespace